src/components/vr_cooperation/include/vr_cooperation/commands/activate_service_request.h

```diff
@@ -33,6 +33,7 @@
 #ifndef SRC_COMPONENTS_VR_COOPERATION_INCLUDE_VR_COOPERATION_COMMANDS_ACTIVATE_SERVICE_REQUEST_H_
 #define SRC_COMPONENTS_VR_COOPERATION_INCLUDE_VR_COOPERATION_COMMANDS_ACTIVATE_SERVICE_REQUEST_H_
 
+#include <string>
```

Comment:
because someone use std::string in this file, and Cpplint complain if not include <string>